src/gfn/gym/bitSequence.py

```diff
@@ -284,7 +284,7 @@ def states_from_tensor(
         """
         if length is None:
             mask = tensor != -1
-            length = mask.int().sum(dim=-1)
+            length = mask.long().sum(dim=-1)
```

Comment:
this became long from int32(?)

Reply:
yes I moved everything to long (I want to unify the type - I'm fine with it being int32).